styles.css

```diff
@@ -0,0 +1,107 @@
+
+* {
+  box-sizing: border-box;
+}
+
+#container {
+  /* width: 375px;
+  height: 710px; */
+  border: 1px solid #e5e5e5;
+  background: #e5e5e5;
+  display: flex;
+  flex-direction: column;
+  margin: 0 auto;
+}
```

Comment:
- [ ] Your project has a wonderful first impression 👍 , but I noticed something you need to work on about your background image 😄 , Kindly make sure the position of your background stick to the Fgigma design , it means it should be positioned exactly as it is in the Figma.

| Your design | Figma design |
| :---: | :---: |
| ![Capture](https://user-images.githubusercontent.com/80458199/234548887-09bf5f71-a2a2-4d34-af7d-cb3beb5d6dd5.JPG) | ![Capture1](https://user-images.githubusercontent.com/80458199/234548927-579d457b-3d4c-42f0-bc92-c534580de086.JPG) |